app/services/merge_requests/update_service.rb

```diff
@@ -23,11 +23,15 @@ def execute(merge_request)
         merge_request.update_nth_task(params[:task_num].to_i, false)
       end
 
+      old_labels = merge_request.labels.map{ |label| label }
```

Comment:
This should also just be `merge_request.labels`. No need for map, it's just creating an array again.